MonoGame.Framework/Windows/WinFormsGamePlatform.cs

```diff
@@ -134,7 +134,7 @@ public override void StartRunLoop()
         
         public override void Exit()
         {
-            Application.Exit();
+            _window.Dispose();
```

Comment:
Yeah. But it is already been done @ https://github.com/KonajuGames/MonoGame/commit/d6a65c8d3a3b86b40366a0466e46f35b17decf64